utils/pre-onnx-mlir.py

```diff
@@ -44,7 +44,7 @@
 
 # Opset version supported by current onnx-mlir
 # Should be consistent with gen_onnx_mlir.py
-current_onnx_mlir_support_version=16
+current_onnx_mlir_support_version=19
```

Comment:
Could you add a stylized comments maybe such as `# ==UPDATE_ALL_VERSION_NUMBER== by scanning for this pattern` in every file where "19" appears (meaning 19 as the current version number) if it shows up in more than one place?

That will make is

Reply:
There's only three that I see. This file, gen_onnx_mlir.py and in OpBuildTable.inc. In gen_onnx_mlir.py, it's coding the latest updated version for each individual Op so using this variable doesn't make sense. OpBuildTable.inc also has `19` in reference to an Opset but that has "Do not modify this file directly. ... This file is automatically generated via script." so also not applicable.

In the spirit of the comment, pre-onnx-mlir.py has `current_onnx_version = "1.14.0"`. I'll see about coming up with some sort of comment tag group these together.